easybuild/easyconfigs/t/TensorFlow/TensorFlow-0.12.1-foss-2016a-Python-2.7.11.eb

```diff
@@ -0,0 +1,66 @@
+easyblock = 'Bundle'
+
+name = 'TensorFlow'
+version = '0.12.1'
+versionsuffix = '-Python-%(pyver)s'
+
+homepage = 'https://www.tensorflow.org/'
+description = """TensorFlow™ is an open source software library for numerical computation using 
+data flow graphs. Nodes in the graph represent mathematical operations, while the graph edges
+represent the multidimensional data arrays (tensors) communicated between them. The flexible
+architecture allows you to deploy computation to one or more CPUs or GPUs in a desktop, server, 
+or mobile device with a single API. TensorFlow was originally developed by researchers and
+engineers working on the Google Brain Team within Google's Machine Intelligence research 
+organization for the purposes of conducting machine learning and deep neural networks research,
+but the system is general enough to be applicable in a wide variety of other domains as well."""
+
+toolchain = {'name': 'foss', 'version': '2016a'}
+
+# this is a bundle of Python packages
+exts_defaultclass = 'PythonPackage'
+use_pip = True
+unpack_sources = False
```

Comment:
@RvDijk please check whether this fixes the problem: https://github.com/hpcugent/easybuild-easyblocks/pull/1089